spec/vcloud/core/vapp_spec.rb

```diff
@@ -154,6 +154,42 @@
       end
     end
 
+  context "power off" do
+    context "successful power off" do
+      before(:each) do
+        @fog_vapp_body = {
+          :name => "Webserver vapp-1",
+          :href => "https://api.vcloud-director.example.com/api/vApp/vapp-63d3be58-2d5c-477d-8410-267e7c3c4a02",
+          :Link => [{
+          :rel => "stopped",
+          :type => "application/vnd.vmware.vcloud.vdc+xml",
+          :href => "https://api.vcloud-director.example.com/api/vdc/074aea1e-a5e9-4dd1-a028-40db8c98d237"
+        }]
+        }
+      end
+
+      it "should power off a vapp that is powered on" do
+        vapp = Vcloud::Core::Vapp.new(@vapp_id)
+        expect(@mock_fog_interface).to receive(:get_vapp).twice().and_return(
+          {:status => Vcloud::Core::Vapp::STATUS::RUNNING},
+          {:status => Vcloud::Core::Vapp::STATUS::POWERED_OFF}
+        )
+        expect(@mock_fog_interface).to receive(:power_off_vapp).with(vapp.id)
+        state = vapp.power_off
+        expect(state).to be_true
+      end
+
+      it "should not power off a vapp that is already stopped, but should return true" do
```

Comment:
Maybe change this test name to exclude everything after the comma, since the functionality being tested is described in the first half?